pulsar-broker/src/main/java/com/yahoo/pulsar/broker/PulsarService.java

```diff
@@ -565,41 +567,56 @@ public MessagingServiceShutdownHook getShutdownService() {
     /**
      * Derive the host
      *
-     * @param isBindOnLocalhost
-     * @return
+     * @return Hostname or IP address the service binds on.
      */
     public static String host(ServiceConfiguration config) {
```

Comment:
this `host(...)` method could probably be removed now, given that we can directly use `config.getBindAddress()` which defaults to a non-null value.